Essentials/src/com/earth2me/essentials/MetaItemStack.java

```diff
@@ -343,14 +344,11 @@ public void addFireworkMeta(final CommandSource sender, final boolean allowShort
     }
 
     private Color getColorFromRgbHexString(String color) {
-        Color c;
         try {
-            c = Color.fromRGB(Integer.decode(color.substring(color.indexOf('[') + 4, color.indexOf(']') - 1)));
-        } catch (Exception e) {
-            c = null;
+            return Color.fromRGB(Integer.decode(color.substring(color.indexOf("[") + 4, color.indexOf("]") - 1)));
```

Comment:
This can throw an `IndexOutOfBoundsException`. You should check that the call to `color.substring()` will succeed before making it.